nucypher/control/emitters.py

```diff
@@ -243,6 +243,14 @@ def __init__(self,
 
         self.log = Logger('web-emitter')
 
+    def _log_exception(self, e, error_message, log_level, response_code):
+        exception = f"{type(e).__name__}: {str(e)}" if str(e) else type(e).__name__
+        message = f"{self} [{str(response_code)} - {error_message}] | ERROR: {exception}"
+        logger = getattr(self.log, log_level)
+        # See #724 / 2156
+        message_cleaned_for_logger = message.replace("{", "<^<").replace("}", ">^>")
```

Comment:
I know your just keeping up with status-quo here, but I think using a utility function in logging.py will make this more readable and less noisy.

Reply:
✔️ - Turns out there was one already.